tests/run-android-arm64.sh

```diff
@@ -1,14 +1,28 @@
 #!/bin/sh
 
 arch=arm64
+device=$1
+
+if [ -z $device ]; then
+  echo "[-] Specify device to run test"
+  echo "$0 <device>"
+  exit 1
+fi
+
+os=$(uname -s)
+if [ $os = "Linux" ]; then
+  os="linux"
+else
+  os="macos"
+fi
```

Comment:
Would be good to move this to a script next to it that we source, so we can re-use this from the other scripts too.

(But maybe better to do this in a separate PR?)

Reply:
will do in a separete PR